src/Language/PureScript/Errors.hs

```diff
@@ -733,10 +737,28 @@ prettyPrintSingleError (PPEOptions codeColor full level showWiki) e = flip evalS
       paras $ [ line "Wildcard type definition has the inferred type "
               , markCodeBox $ indent $ typeAsBox ty
               ] ++ renderContext ctx
-    renderSimpleErrorMessage (HoleInferredType name ty ctx) =
+    renderSimpleErrorMessage (HoleInferredType name ty ctx (TSBefore _)) =
       paras $ [ line $ "Hole '" ++ markCode name ++ "' has the inferred type "
-              , markCodeBox $ indent $ typeAsBox ty
+              , markCodeBox (indent (typeAsBox ty))
               ] ++ renderContext ctx
+    renderSimpleErrorMessage (HoleInferredType name ty ctx (TSAfter idents)) =
+      paras $ [ line $ "Hole '" ++ markCode name ++ "' has the inferred type "
+              , markCodeBox (indent (typeAsBox ty))
+              ]
+              ++ let formatTS (names, types) =
+                        let
+                          idBoxes = Box.text . showQualified runIdent <$> names
+                          tyBoxes = (\t -> BoxHelpers.indented (Box.text ":: " Box.<> typeAsBox t)) <$> types
+                          longestId = maximum (map Box.cols idBoxes)
+                        in
+                          Box.vcat Box.top $
```

Comment:
Oh, of course. Could you please update the PR description to show how this looks now?